code/script.js

```diff
@@ -1,47 +1,110 @@
-// All the DOM selectors stored as short variables
-const chat = document.getElementById('chat')
+// DOM SELECTORS:
+const chat = document.getElementById("chat");
+const form = document.getElementById("name-form");
+const nameInput = document.getElementById("name-input");
+const inputWrapper = document.getElementById("input-wrapper");
 
-// Global variables, if you need any, declared here
+// GLOBAL VARIABLE:
+let currentQuestion = 1;
 
-// Functions declared here
+// FUNCTIONS:
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// 1rs Function showMessage: This function will add a chat bubble in the correct place based on who the sender is.
 const showMessage = (message, sender) => {
-  if (sender === 'user') {
+  if (sender === "user") {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="./Pictures/bill-chatbot.png" alt="User" />  
       </section>
-    `
-  } else if (sender === 'bot') {
+    `;
+  } else if (sender === "bot") {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="./Pictures/julieta-chatbot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
-    `
+    `;
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
-}
+  chat.scrollTop = chat.scrollHeight; // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+};
 
-// Starts here
+// 2nd Function greeting: This function invokes ('calls') the first function, so when the function greeting is invoked, it invokes the first function.
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Welcome to Flowerland! What's your name?`, "bot");
+};
+setTimeout(greeting, 1000);
+
+// 3rd Function handleInput
+const handleInput = (event, answer) => {
+  event.preventDefault();
+
+  if (currentQuestion === 1) {
+    let name = nameInput.value;
+    showMessage(name, "user");
+    setTimeout(() => showFlowerTypes(name), 1000);
+
+  } else if (currentQuestion === 2) {
+    showMessage(answer, "user");
+    inputWrapper.innerHTML = "";
+    setTimeout(() => showFlowerSize(answer), 2000);
```

Comment:
perhaps this delay of 2000 is too long, may be a little quicker response (like 500or 1000) would be more user friendly. As now it seems that there is something wrong with the bot for a while.